tools/ops/mugc.py

```diff
@@ -148,8 +155,14 @@ def setup_parser():
         '-c', '--config', dest="config_files", nargs="*", action='append',
         help="Policy configuration files(s)", default=[])
     parser.add_argument(
-        "--present", action="store_true", default=False,
```

Comment:
again, this is a breaking change and doesnt appear to really make a large difference in the script